test/conformance/conformance_test.go

```diff
@@ -87,7 +87,7 @@ func TestGatewayConformance(t *testing.T) {
 	// Currently mode only relies on the KongRouterFlavor, but in the future
 	// we may want to add more modes.
 	mode := string(config.KongRouterFlavor)
-	reportFileName := fmt.Sprintf("kong-gateway-operator-%s.yaml", mode) // TODO: https://github.com/Kong/gateway-operator/issues/268
+	reportFileName := fmt.Sprintf("standard-%s-%s-report.yaml", metadata.Release, mode)
```

Comment:
Looking at upstream reports e.g. https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports/v1.1.0/istio-istio I can see that they have the following format `${version}-report.yaml`. Why include the `standard` here?

Reply:
this one uploaded by istio is not properly formatted. Here are the naming rules: https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports

Reply:
😱 

Ok, good to know. 👍

Reply:
I've submitted a PR upstream to fix it https://github.com/kubernetes-sigs/gateway-api/pull/3148